config/gradle/jre.gradle

```diff
@@ -15,8 +15,8 @@
  */
 
 // Uses Bellsoft Liberica JRE
-def jreVersion = '8u212'
-def jreUrlBase = "https://download.bell-sw.com/java/${jreVersion}/bellsoft-jre${jreVersion}"
+def jreVersionGame = '8u212'
```

Comment:
I think we checked that we can also run the game with JRE  11, thus I would like to get rid of this and use the same runtime for both.